src/domain/app/App.css

```diff
@@ -0,0 +1,22 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #09d3ac;
+}
```

Comment:
In other projects, we're using `SASS` modules, I wonder whether you'd like to consider it as well.

Reply:
Yes that was the plan. Updated in the latest commits.